examples/terraform/main.tf

```diff
@@ -31,6 +31,11 @@ resource "aws_lambda_function" "hello_world" {
   memory_size   = 1024 # <<<<< Try changing this to 512 to compare costs
 }
 
+moved {
+  from = aws_lambda_function.hola_mundo
+  to   = aws_lambda_function.hello_world
+}
+
```

Comment:
```suggestion
```

can we remove this from the `examples/terraform` file as this is used as a getting started point for many users.

Perhaps we can add this moved block to `cmd/infracost/testdata/breakdown_multi_project_autodetect/dev/main.tf` instead as this will be covered by one of the breadkdown tests.

But in this case it would need to be >

```
moved {
  from = aws_instance.foo
  to   = aws_lambda_function.web_app
}
```